cli/src/main/scala/com/advancedtelematic/tuf/cli/repo/TufRepo.scala

```diff
@@ -68,15 +68,31 @@ object TufRepo {
 
   case class RootPullError(errors: NonEmptyList[String]) extends Exception("Could not validate a valid root.json chain:\n" + errors.toList.mkString("\n")) with NoStackTrace
 
-  protected [cli] def readConfigFile(repoPath: => Path): Try[RepoConfig] =
+  protected [cli] def readConfig(repoPath: => Path): Try[RepoConfig] =
     Try { new FileInputStream(repoPath.resolve("config.json").toFile) }
       .flatMap { is => CliUtil.readJsonFrom[RepoConfig](is) }
 
-  protected [repo] def writeConfigFiles(repoPath: Path, repoUri: URI, treehubConfig: TreehubConfig,
-                                        authConfig: Option[AuthConfig], repoServerType: TufServerType): Try[RepoConfig] = Try {
-    val repoConfig = RepoConfig(repoUri, authConfig, treehubConfig, repoServerType)
-    Files.write(repoPath.resolve("config.json"), repoConfig.asJson.spaces2.getBytes)
-    repoConfig
+  def importTlsCerts(repoPath: Path, clientCert: Path, serverCert: Option[Path]): Try[RepoConfig] = {
+    for {
+      oldConfig <- readConfig(repoPath)
+      newConfig <- Try {
+        val clientCertPath = repoPath.resolve("client-auth.p12")
```

Comment:
Since the `autoprov_credentials.p12` contains an underscore i would prefer an underscore here as well.